examples/mux-selftest.yaml

```diff
@@ -28,15 +44,16 @@ hw:
     corruptlist: ['upper_node_list']
 distro: !mux     # This node is set as !multiplex below
     fedora:
+        !filter-out : "/virt/hw/disk/scsi"
         init: 'systemd'
 env: !mux
     opt_CFLAGS: '-Os'
     prod:
         opt_CFLAGS: 'THIS SHOULD GET OVERWRITTEN'
 env: !mux
+    !filter-out : "/yet/another/nonexisting/node"  # let's see if filters are updated when merging
     prod:
         opt_CFLAGS: '-O2'
 distro: !mux
     mint:
         init: 'systemv'
```

Comment:
This one was always only for selftests, but sure, we can move it to `selftests/.data` and provide a clean example in some cleanup PR.